docs/tutorials/autoscaling-federatedhpa-with-cronfederatedhpa.md

```diff
@@ -0,0 +1,189 @@
+  selector:
+    matchLabels:
+      app: nginx
+  template:
+    metadata:
+      labels:
+        app: nginx
+    spec:
+      containers:
+      - image: nginx
+        name: nginx
+        resources:
+          requests:
+            cpu: 25m
+            memory: 64Mi
+          limits:
+            cpu: 25m
+            memory: 64Mi
+---
+apiVersion: policy.karmada.io/v1alpha1
+kind: PropagationPolicy
+metadata:
+  name: nginx-propagation
+spec:
+  resourceSelectors:
+    - apiVersion: apps/v1
+      kind: Deployment
+      name: nginx
+  placement:
+    clusterAffinity:
+      clusterNames:
+        - member1
+        - member2
+    replicaScheduling:
+      replicaDivisionPreference: Weighted
+      replicaSchedulingType: Divided
+      weightPreference:
+        staticWeightList:
+          - targetCluster:
+              clusterNames:
+                - member1
+            weight: 1
+          - targetCluster:
+              clusterNames:
+                - member2
+            weight: 1
+```
+
+After deploying, you can check the created pods:
+```sh
+$ karmadactl get pods
+NAME                     CLUSTER   READY   STATUS    RESTARTS   AGE
+nginx-777bc7b6d7-cmt6k   member1   1/1     Running   0          27m
+```
+
+## Deploy FederatedHPA in Karmada control plane
+
+Let's create a FederatedHPA in the Karmada control plane to manage the cross-cluster nginx deployment:
+```yaml
+apiVersion: autoscaling.karmada.io/v1alpha1
+kind: FederatedHPA
+metadata:
+  name: nginx-fhpa
+spec:
+  scaleTargetRef:
+    apiVersion: apps/v1
+    kind: Deployment
+    name: nginx
+  minReplicas: 1
+  maxReplicas: 10
+  behavior:
+    scaleDown:
+      stabilizationWindowSeconds: 10
+    scaleUp:
+      stabilizationWindowSeconds: 10
+  metrics:
+    - type: Resource
+      resource:
+        name: cpu
+        target:
+          type: Utilization
+          averageUtilization: 80
+```
+## Deploy CronFederatedHPA in Karmada control plane
+
+To autoscale the minReplicas of FederatedHPA, let's create CronFederatedHPA in the Karmada control plane:
+```yaml
+apiVersion: autoscaling.karmada.io/v1alpha1
+kind: CronFederatedHPA
+metadata:
+  name: nginx-cronfhpa
+  namespace: default
+spec:
+   scaleTargetRef:
+      apiVersion: autoscaling.karmada.io/v1alpha1
+      kind: FederatedHPA
+      name: nginx-fhpa
+   rules:
+   - name: "scale-up"
+     schedule: "*/1 * * * *"
+     targetMinReplicas: 5
+```
+
+The `spec.schedule` follows the following format:
+```
+# ┌───────────── minute (0 - 59)
+# │ ┌───────────── hour (0 - 23)
+# │ │ ┌───────────── day of the month (1 - 31)
+# │ │ │ ┌───────────── month (1 - 12)
+# │ │ │ │ ┌───────────── day of the week (0 - 6) (Sunday to Saturday;
+# │ │ │ │ │                                   7 is also Sunday on some systems)
+# │ │ │ │ │                                   OR sun, mon, tue, wed, thu, fri, sat
+# │ │ │ │ │
+# * * * * *
+```
+The expression `*/1 * * * *` indicates that the FederatedHPA should be scaled every minute.
```

Comment:
The expression indicates the `periodical time` to `evaluate` the replica instead of `scale action`.
I know you are trying the explain the same thing, my opinion is just to make it clear.